voxxed-microservices-2018/module-01.adoc

```diff
@@ -249,14 +241,14 @@ For building a graphical dashboard with such information, it's possible to use G
 The Prometheus service pod runs with `prometheus-server` service account and it needs to have access to the API server to get the pod list and for allowing that, the following command is needed.
 
 [source,sh]
-$ export NAMESPACE=l1099-kafka
+$ export NAMESPACE=voxxed
 $ oc create sa prometheus-server
 $ oc adm policy add-cluster-role-to-user cluster-reader system:serviceaccount:${NAMESPACE}:prometheus-server
 
-Finally, create the Prometheus service by running.
+Finally, create the Prometheus service by running:
 
 [source,sh]
-$ oc create -f https://raw.githubusercontent.com/strimzi/strimzi/master/metrics/examples/prometheus/kubernetes.yaml
+$ oc apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/master/metrics/examples/prometheus/kubernetes.yaml
 
 ==== Grafana
```

Comment:
That's not quite working yet, I don't see any data points in Grafana. Still need to get to the ground of it.

Reply:
Did you checked Prometheus if it contains the data points? If Prometheus doesn't see them, then the problem will be in Prom configuration which doesn't work and collect the metrics.